presto-main/src/test/java/io/prestosql/operator/scalar/time/TestTime.java

```diff
@@ -1660,6 +1660,7 @@ public void testAtTimeZone()
     {
         Session session = assertions.sessionBuilder()
                 .setTimeZoneKey(TimeZoneKey.getTimeZoneKey("Pacific/Apia"))
+                .setStart(Instant.from(ZonedDateTime.of(2020, 5, 1, 12, 0, 0, 0, assertions.getDefaultSession().getTimeZoneKey().getZoneId())))
```

Comment:
use same zone as used for `setTimeZoneKey` instead of `assertions.getDefaultSession().getTimeZoneKey()`

(or use UTC if you want do not want to have the same)